src/backend/vulkan/src/window.rs

```diff
@@ -16,13 +16,102 @@ use winit;
 use conv;
 use {VK_ENTRY, Backend, Instance, PhysicalDevice, QueueFamily, RawInstance};
 
+pub trait GetExtentable {
+    fn get_extent(&self) -> Option<(u32, u32)>;
```

Comment:
any reason this is not `hal::window::Extent2D`?

Reply:
No

Reply:
heh, I was reading it over email thinking, hmmm... did I ask "can we use `Extend2D`?" and you just replied straight "No" :rofl: